pkgs/development/tools/misc/autobuild/default.nix

```diff
@@ -1,10 +1,11 @@
 { fetchurl, lib, stdenv, makeWrapper, perl, openssh, rsync }:
 
 stdenv.mkDerivation rec {
-  name = "autobuild-5.3";
+  pname = "autobuild";
+  version = "5.3";
 
   src = fetchurl {
-    url = "http://savannah.spinellicreations.com/autobuild/${name}.tar.gz";
+    url = "http://savannah.spinellicreations.com/${pname}/${pname}-${version}.tar.gz";
```

Comment:
Name is not resoling. Can we use any other savannah mirror here or the mirror:// style url?